dspace-server-webapp/src/test/java/org/dspace/app/rest/builder/BitstreamBuilder.java

```diff
@@ -152,7 +152,15 @@ public Bitstream build() {
 
     @Override
     public void cleanup() throws Exception {
-        delete(bitstream);
+       try (Context c = new Context()) {
+            c.turnOffAuthorisationSystem();
+            // Ensure object and any related objects are reloaded before checking to see what needs cleanup
+            bitstream = c.reloadEntity(bitstream);
+            if (bitstream != null) {
+                delete(c, bitstream);
+                c.complete();
```

Comment:
seems a risk to keep the context open?

Reply:
it uses the try-with-resource syntax (it is used also in other recent part of the sourcecode)
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html